packages/message-box/src/main.vue

```diff
@@ -25,9 +25,9 @@
             <i class="el-message-box__close el-icon-close"></i>
           </button>
         </div>
-        <div class="el-message-box__content" v-if="message !== ''">
+        <div class="el-message-box__content">
           <div class="el-message-box__status" :class="[ typeClass ]" v-if="typeClass && !center"></div>
```

Comment:
Please also hide this `<div>` when `message` is empty.

Reply:
add v-show and v-if ?

Reply:
`v-if="typeClass && !center && message !== ''"`

Reply:
do you mean this

```
<div class="el-message-box__content" v-if="typeClass && !center && message !== ''">
``` 

how can i hide this div ? what am I trying to do is show `el-message-box__input` even `options.message` is empty when `options.showInput` is true.

maybe this code below works. 

```
<div class="el-message-box__content" v-if="(typeClass && !center) || message !== ''" || showInput>
```

but I think it's unnecessary?

tell me if i was wrong

Reply:
Change line29 from
```html
<div class="el-message-box__status" :class="[ typeClass ]" v-if="typeClass && !center"></div>
```
to
```html
<div class="el-message-box__status" :class="[ typeClass ]" v-if="typeClass && !center && message !== ''"></div>
```

My comment is right under line29.

Reply:
sorry, did't notice that.